CODEOWNERS

```diff
@@ -0,0 +1,51 @@
+# This file contains information about the code owners for various
+# files in the MRTK codebase.
+#
+# For more information on CODEOWNERS, see:
+# https://help.github.com/en/articles/about-code-owners
+#
+# For syntax, see:
+# https://help.github.com/en/articles/about-code-owners#codeowners-syntax
+#
+# The intent of this file is to list, on a per-file/per-area basis:
+# 1) Who is the most familiar with an area of code.
+# OR
+# 2) Who can take a first pass at pull request review, or delegate the review
+# to someone else who may be more appropriate.
+#
+# Note that order in this file is EXTREMELY important - the last matching
+# rule will win. This means that the most general rules should be listed
+# first in this file, followed by more specific rules.
+
+# Build / CI Pipeline
+/Pipelines/ @keveleigh
+
+# Checked-in external dependencies
+/ExternalDependencies/ @keveleigh @davidkline-ms
+
+# Tooling
+/Tooling/ @keveleigh @Zee2
+
+# Packaging
+*.asmdef @keveleigh @davidkline-ms
+package.json @davidkline-ms @keveleigh

```

Comment:
I'm not sure if these will be hit, for the package folders listed below at least.

> Note that order in this file is EXTREMELY important - the last matching
> rule will win. This means that the most general rules should be listed
> first in this file, followed by more specific rules.

Reply:
interesting. this was the order in the file from the main branch. happy to move to the bottom.

Reply:
moved to the end of the file.